pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -185,6 +204,40 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 	)
 }
 
+func (i *Gatherer) gatherNamespaceEvents(namespace string) ([]record.Record, []error) {
+	// do not accidentally collect events for non-openshift namespace
+	if !strings.HasPrefix(namespace, "openshift-") {
+		return []record.Record{}, nil
+	}
+	events, err := i.coreClient.Events(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return nil, []error{err}
+	}
+	// filter the event list to only recent events
+	now := time.Now()
+	var filteredEventIndex []int
+	for i := range events.Items {
+		if events.Items[i].LastTimestamp.Time.Before(now.Add(-1 * maxEventTimeInterval)) {
```

Comment:
do this calculation earlier (not in the loop)

Reply:
also `- maxEventTimeInterval` is clearer

Reply:
i don't know what you mean @smarterclayton with the calculation, in that loop I only pick events that are matching that condition (maxEventTimeInterval) then I build slice with the size  matching the number of filtered events and fill that slice with that events... I think it is already efficient?

Reply:
Don't do `now.Add(-1 *x)` in the loop.  Calculate that up front `oldestEvent := ...` and then make this `...Time.Before(oldestEvent)`